src/m3ninx/idx/query.go

```diff
@@ -34,6 +34,13 @@ func (q Query) String() string {
 	return q.query.String()
 }
 
+// NewQueryFromSearchQuery creates a new Query from a search.Query.
+func NewQueryFromSearchQuery(q search.Query) Query {
```

Comment:
Would it be better to have this as a helper in `block_prop_test.go` if it's not used elsewhere?

Reply:
different package unfortunately :/ Otherwise I wouldn't need the constructor at all

Reply:
Ah my bad, missed that you were setting the private `query` field